discovery-provider/src/queries/notifications.py

```diff
@@ -1164,10 +1165,30 @@ def solana_notifications():
                 }
             )
 
+        reaction_results: List[Reaction] = (
+            session.query(Reaction)
+            .filter(Reaction.slot >= min_slot_number, Reaction.slot <= max_slot_number)
+            .all()
+        )
+        reactions = []
+        for reaction in reaction_results:
+            reactions.append(
+                {
+                    const.solana_notification_type: const.solana_notification_type_reaction,
+                    const.solana_notification_slot: reaction.slot,
+                    const.notification_initiator: reaction.sender_wallet,
```

Comment:
Do we want `sender_wallet` or `sender_user_id` here?

Reply:
gooood call - should be user_id